Automattic/wp-calypso
# Display a "purchase success" notice in Customer Home

#### Changes proposed in this Pull Request

* When a new user lands in Customer Home after successfully purchasing a plan, show a success notice. Otherwise there's no indication that the transaction was successful.
* A minor UI fix that removes the space before "/month" when displaying price in checkout.

**New signup -> Checkout -> Customer Home**

<img width="1250" alt="Screenshot 2021-01-29 at 1 04 03 PM" src="https://user-images.githubusercontent.com/1269602/106245475-825dbc00-6232-11eb-833b-276453d86ee0.png">

**Launch flow -> Checkout -> Customer Home**

<img width="1235" alt="Screenshot 2021-01-29 at 12 45 55 PM" src="https://user-images.githubusercontent.com/1269602/106245519-92759b80-6232-11eb-8f33-7ad8865936a2.png">


#### Testing instructions



* Signup as a new user, purchase a plan, decline any upsell offers. Verify that you are taken to Customer Home with a success notice as shown in the screenshots above.
* Enter the site launch flow, purchase a plan, and verify that you are taken to Customer Home with a success notice.
* Signup as a new user, select a free plan, and confirm that you _do not see_ any notices when on Customer Home.
* Repeat step 1 with a non-EN language and verify that the success notice appears translated:
<img width="1264" alt="Screenshot 2021-01-29 at 12 58 21 PM" src="https://user-images.githubusercontent.com/1269602/106245375-5b06ef00-6232-11eb-890e-47f04a696624.png">
* Verify that in the checkout page, the price is displayed as "$4/month" where there's no space before 
"/month". Verify that in non-EN, translations are not affected.

<img width="571" alt="Screenshot 2021-02-03 at 3 59 45 PM" src="https://user-images.githubusercontent.com/1269602/106734321-e6202480-6638-11eb-8478-b93d35b795fe.png">
